docs/layouts/shortcodes/generated/yarn_config_configuration.html

```diff
@@ -182,5 +182,15 @@
             <td>String</td>
             <td>Specify YARN node label for the Flink TaskManagers, it will override the yarn.application.node-label for TaskManagers if both are set.</td>
         </tr>
+        <tr>
+          <td><h5>yarn.view.acls</h5></td>
```

Comment:
Should we link to either this [doc](https://docs.cloudera.com/documentation/enterprise/latest/topics/cm_mc_yarn_acl.html) or this [doc](https://docs.cloudera.com/runtime/7.2.7/yarn-security/yarn-security.pdf)? Looks like there are more ACL rules and syntax. For eg:
1. Group names are case sensitive.
2. If an ACL starts with a single space, then it must consist of groups only.

Reply:
@venkata91 Can you please suggest on the process to update the doc.

Reply:
Thanks for sharing the other doc. I feel we should use the [doc](https://docs.cloudera.com/runtime/7.2.7/yarn-security/yarn-security.pdf). Please let me know your thoughts.